cpp/daal/include/services/internal/sycl/math/lapack_executor.h

```diff
@@ -137,10 +138,14 @@ class PotrsExecutor
 
 public:
     static void run(cl::sycl::queue & queue, const math::UpLo uplo, const size_t n, const size_t ny, UniversalBuffer & a_buffer, const size_t lda,
-                    UniversalBuffer & b_buffer, const size_t ldb, services::Status & status)
+                    UniversalBuffer & b_buffer, const size_t ldb, Status & status)
     {
-        Execute op(queue, uplo, n, ny, a_buffer, lda, b_buffer, ldb, status);
-        TypeDispatcher::floatDispatch(a_buffer.type(), op);
+        DAAL_ASSERT(!a_buffer.empty());
+        DAAL_ASSERT(!b_buffer.empty());
+        DAAL_ASSERT(a_buffer.type() == b_buffer.type());
+
```

Comment:
Size check?

Reply:
Size check is going to be performed in Executor